crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -834,9 +835,7 @@ pub enum GpuBufferInfo {
         count: u32,
```

Comment:
The vertex count would be stored twice (once in `GpuMesh.vertex_count` field, once in `GpuMesh.buffer_info.Indexed.count`) With this solution.

Why not define `vertex_count()` as a method on `GpuMesh`? This way you inspect the content of `buffer_info` and return the value of either `NonIndex.vertex_count` or `Indexed.count`.

Reply:
The count stored there is the size of the index buffer, not the amount of vertices in the mesh

Reply:
The same thing confused me at first. Imo, we should rename Indexed.count to Indexed.indices_count or smth.

Reply:
`size` would be appropriate then